docs/Makefile

```diff
@@ -7,6 +7,7 @@ SPHINXOPTS    ?=
 SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
+GIT_COMMIT    ?= $(shell git rev-parse HEAD)
```

Comment:
could you share what this change related to, is this the docs?

Reply:
This is for the docs, basically we currently are linking to hard-coded git hashs (which basically means we're linking in our current code to old code, which is a live link so it passes our CI checks, but it's actually an invalid link), or to the main branch, which will be an issue if we want to build our docs for a release branch.